pandas/parser.pyx

```diff
@@ -563,6 +563,18 @@ cdef class TextReader:
                 else:
                     raise ValueError('Python 2 cannot read bz2 from open file '
                                      'handle')
+            elif self.compression == 'zip':
+                import zipfile
```

Comment:
#11666 allows gz/bz in pickles. That's next on the pipeline.

#11667 I'm not sure how this is relevant